src/loaders/Loader.d.ts

```diff
@@ -17,6 +17,8 @@ export class Loader {
 	parse(): void;
 	*/
 
+	loadAsync( url: string, onProgress?: ( event: ProgressEvent ) => void ): Promise<any>;
```

Comment:
This should be `Promise<void>` ? https://github.com/mrdoob/three.js/pull/19118/files#diff-eddb05b2beb45b0b964af1db92ad413dR25 does not return.

Reply:
The value eventually passed into `resolve` should parameterize the Promise. For example TextureLoader.prototype.loadAsync would return `Promise<Texture>`. But on the abstract class we don't know the resolved type.

Reply:
Ah, I'm blind, you're right!